ConnectorCSI/ConnectorCSIShared/UI/ConnectorBindingsCSI.Send.cs

```diff
@@ -148,12 +175,26 @@ public override async Task<string> SendStream(StreamState state, ProgressViewMod
       if (progress.Report.OperationErrorsCount != 0)
         return null;
 
+      if (branchName != null)
+      {
+        var branchesSplit = state.BranchName.Split('/');
+        branchesSplit[branchesSplit.Count() - 1] = branchName;
+        branchName = string.Join("", branchesSplit);
```

Comment:
why is this needed?

Reply:
that was sort of left over from the structural diffing that I did. I was sending to a "report" branch and I wanted it to have the same nesting as the branch that I was receiving from. I figure we may use that functionality eventually so I would just leave it. Let me know and I can get rid of that.